crates/valence_nbt/src/compound.rs

```diff
@@ -41,11 +41,11 @@ impl fmt::Debug for Compound {
 }
 
 impl Compound {
```

Comment:
`new` and `with_capacity` cannot be `const` because `Map::new` and `Map::with_capacity` are not always `const`.